cli/gitfs.ts

```diff
@@ -49,11 +49,11 @@ export async function uploadRefs(id: string, repoUrl: string): Promise<void> {
     let visited: SMap<boolean> = {};
     let toCheck: string[] = [];
 
-    await processCommit(id);
+    await processCommit(initialCommitId);
 
     await uploadMissingObjects(undefined, true);
 
-    await refreshRefs(id, repoUrl);
+    await refreshRefs(initialCommitId);
```

Comment:
parameter gone?

Reply:
I saw you resolved this conversation, but to clarify for anyone else the dropped parameter was just redundant with the enclosing fns parameter; did that in a few places where the same was true or the parameter was never used